### PR TITLE
Feat/fe task detail and history

### PR DESCRIPTION
**Pull request by prem056627**
Added : Task history modal and text changes

### CODE REVIEW

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10466327644). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

File: frontend/src/pages/appPoliciesManagement/components/AppTable/RowMenu.jsx [hunk 1]
```diff
@@ -73,30 +73,6 @@ export default function RowMenu({ rowData }) {
...
-											Delete Policy
-										</span>
-										<span className="text-start font-lato text-[12px] leading-[16px] tracking-[0.2px] text-[#6C747D]">
-											this will delete the entire permission along with all it’s
-											details
-										</span>
-									</div>
-								</button>
-							)}
-						</Menu.Item> */}
```
Comment (sukuna28) [inline frontend/src/pages/appPoliciesManagement/components/AppTable/RowMenu.jsx:99]: @prem056627 can you pls remove this commented code?

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10466549643). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

File: frontend/src/pages/appReleasesRoutes/routes/Index.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { Navigate, Route, Routes } from 'react-router-dom';
+// import AppLogs from '../components/appLogs/Index';
+import Releases from '../components/appLogs/Index';
+// import AppAccessLogs from '../components/AppAccessLogs/index.js';
+// import AppLogs from '../components/AppLogs.jsx';
```
Comment (sukuna28) [inline frontend/src/pages/appReleasesRoutes/routes/Index.jsx:5]: @prem056627 remove unused imports

File: frontend/src/pages/appPackagesManagement/components/AppTable/RowMenu.jsx [hunk 3]
```diff
@@ -94,9 +90,6 @@ export default function RowMenu({ rowData }) {
 											<span className="text-start font-lato text-[14px] font-bold leading-[20px] tracking-[0.2px] text-[#212429]">
 												Configure Package
 											</span>
-											{/* <span className="text-start font-lato text-[12px] leading-[16px] tracking-[0.2px] text-[#6C747D]">
-												configure
-											</span> */}
```
Comment (sukuna28) [inline frontend/src/pages/appPackagesManagement/components/AppTable/RowMenu.jsx:99]: @prem056627 remove commnets

File: frontend/src/pages/app/routes/index.js [hunk 4]
```diff
@@ -19,6 +19,8 @@ import { selectAppPanelInitialData } from '../../platform/slice';
 import Chatbot from '../components/Chatbot';
 import DragablePopover from '../components/Chatbot/DragablePopover';
 import SideMenu from '../components/SideMenu';
+import { AppLogsRoutes, AppReleasesRoutes } from '../../appReleasesRoutes/routes/Index';
+// import { AppLogsRoutes } from '../../appReleasesRoutes/components/appLogs/Index'
```
Comment (sukuna28) [inline frontend/src/pages/app/routes/index.js:23]: @prem056627 remove unused import

File: frontend/src/mocks/appTasksManagementHandlers.js [hunk 5]
```diff
@@ -123,6 +126,113 @@ export const appTasksManagementHandlers = [
...
+						is_deleted: true,
+						args: '["CRMApp2", "accounts.tasks.test"]',
+						kwargs: {},
+						interval: null,
+						master_task: 52,
+					},
+				},
+			})
+		);
+	}),
```
Comment (sukuna28) [inline frontend/src/mocks/appTasksManagementHandlers.js:235]: @prem056627 revert this data as previously discussed

File: frontend/src/mocks/appReleasesHandlers.js [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+			// 					"id": "failed",
+			// 					"label": "Failed"
+			// 				}
+			// 			]
+			// 		},
+			// 		"message": "Releases fetched successfully"
+			// 	}
+			// }
+			
+
```
Comment (sukuna28) [inline frontend/src/mocks/appReleasesHandlers.js:136]: @prem056627 remove commented code

File: backend/src/zango/assets/app_panel/js/build.1723531926721.min.js.LICENSE.txt
Comment (sukuna28) [inline backend/src/zango/assets/app_panel/js/build.1723531926721.min.js.LICENSE.txt:1]: @prem056627 remove this file

**Review by sukuna28 — CHANGES_REQUESTED** [review]

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10468662927). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10469415457). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10483477013). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10483605159). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

**Review by sukuna28 — CHANGES_REQUESTED** [review]

File: backend/src/zango/assets/app_panel/js/build.1723531926721.min.js.LICENSE.txt
Comment (shahharsh176) [inline backend/src/zango/assets/app_panel/js/build.1723531926721.min.js.LICENSE.txt:1]: remove this file

File: frontend/src/metadata.json [hunk 7]
```diff
@@ -1 +1 @@
-{ "buildMajor": 0, "buildMinor": 3, "buildPatch": 0, "buildTag": "" }
+{"buildMajor":0,"buildMinor":3,"buildPatch":1,"buildTag":""}
```
Comment (shahharsh176) [inline frontend/src/metadata.json:1]: revert changes in this file

**Review by shahharsh176 — CHANGES_REQUESTED** [review]

**Comment by kc-diabeat** [comment]
The build artifact is available in the artifact section [here](https://github.com/Healthlane-Technologies/Zango/actions/runs/10696533749). You can download it and place the build file under backend/src/zango/assets/app_panel/js for local testing and verification.

File: backend/src/zango/assets/app_panel/js/build.1723531926721.min.js.LICENSE.txt
Comment (shahharsh176) [inline backend/src/zango/assets/app_panel/js/build.1723531926721.min.js.LICENSE.txt:1]: @prem056627 Please remove this file

**Review by shahharsh176 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**